ci/other/pylintrc

```diff
@@ -82,7 +82,47 @@ enable=indexing-exception,old-raise-syntax
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,no-member,no-name-in-module,import-error,unsubscriptable-object,unbalanced-tuple-unpacking,undefined-variable,protected-access,superfluous-parens,invalid-name,no-else-return,useless-super-delegation,len-as-condition,invalid-unary-operand-type
+disable=
+    design,
+    similarities,
+    no-self-use,
+    attribute-defined-outside-init,
+    locally-disabled,
+    star-args,
+    pointless-except,
+    bad-option-value,
+    global-statement,
+    fixme,
+    suppressed-message,
+    useless-suppression,
+    locally-enabled,
+    no-member,
+    no-name-in-module,
+    import-error,
+    unsubscriptable-object,
+    unbalanced-tuple-unpacking,
+    undefined-variable,
```

Comment:
This could be moved to the enable section. As of now, make lint checks only python/mxnet folder - there are no undefined name errors in that folder AFAIK - so it should not break CI.

Reply:
I am unclear what you mean by __moved to the enable section__.  I know little about PyLint.

Reply:
undefined-variable errors were not being caught because the option is disabled by default. Moving this option to line 74 of pylintrc will ensure that it is checked in future builds.

Reply:
Thanks for the explanation... Done.

Reply:
Thanks.. undefined-variable can be removed from the disable section (now that it is in the enable section)